pkgs/tools/networking/mac-telnet/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, gettext
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mac-telnet";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "haakonnessjoen";
+    repo = "MAC-Telnet";
+    rev = "v${version}";
+    hash = "sha256-ZBJ3GTTXV14DMcKZWkhNDfOzsL+cCahBzTtnJsRvw/w=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  buildInputs = [
+    gettext
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+
+    # Remove unnecessary chmodding from Makefile
+    head -1 config/Makefile.am > config/Makefile.am
```

Comment:
This can bitrot in the future. How about replacing that file with an empty file? Or deleting it? Or writing a patch? Or perhaps:

```sh
substituteInPlace config/Makefile.am \
  --replace chown echo \
  --replace chmod echo
```

Reply:
What @doronbehar said but with `--replace-fail`